test/scalacheck/scala/reflect/quasiquotes/UnliftableProps.scala

```diff
@@ -73,8 +73,8 @@ object UnliftableProps extends QuasiquoteProperties("unliftable") {
   }
 
   property("unlift scala.symbol") = test {
-    val q"${s: scala.Symbol}" = q"'foo"
-    assert(s.isInstanceOf[scala.Symbol] && s == 'foo)
+    val q"${s: scala.Symbol}" = q"""sym"foo""""
+    assert(s.isInstanceOf[scala.Symbol] && s == sym"foo")
```

Comment:
There's no way to make this reflection test of symbols to work:

```
! unliftable.unlift scala.symbol: Exception raised on property evaluation.
> Exception: scala.MatchError: StringContext("foo").sym() (of class scala.reflect.internal.Trees$Apply)
scala.reflect.quasiquotes.UnliftableProps$.$anonfun$new$26(UnliftableProps.scala:76)
```
Probably not a big loss?

Reply:
Actually, it looks like this is how utest implements symbol-based test names:

https://github.com/lihaoyi/utest/blob/v0.6.6/utest/shared/src/main/scala/utest/Tests.scala#L33-L39

Fortunately, utest already supports a string-based alternative.